services/121-service/src/queues-registry/register-processor.decorator.ts

```diff
@@ -0,0 +1,21 @@
+import { Processor } from '@nestjs/bull';
+
+const ProcessorRegistry = new Set<string>();
```

Comment:
Maybe rename this to `registeredProcessors`? And just use the value directly instead of converting to an array first.

Reply:
done